libs/ml/tests/unit/ops/tanh.cpp

```diff
@@ -51,22 +52,23 @@ TYPED_TEST(TanHTest, forward_all_positive_test)
 
   for (uint64_t i(0); i < n; ++i)
   {
-    data.Set(i, typename TypeParam::Type(dataInput[i]));
-    gt.Set(i, typename TypeParam::Type(gtInput[i]));
+    data.Set(i, fetch::math::AsType<DataType>(dataInput[i]));
+    gt.Set(i, fetch::math::AsType<DataType>(gtInput[i]));
   }
 
   fetch::ml::ops::TanH<TypeParam> op;
 
   TypeParam prediction(op.ComputeOutputShape({std::make_shared<const TypeParam>(data)}));
   op.Forward({std::make_shared<const TypeParam>(data)}, prediction);
 
-  ASSERT_TRUE(
-      prediction.AllClose(gt, typename TypeParam::Type(1e-4), typename TypeParam::Type(1e-4)));
+  ASSERT_TRUE(prediction.AllClose(gt, fetch::math::AsType<DataType>(1e-4),
```

Comment:
these can be string constructions on 0.0001

Reply:
fixed